File: examples/ruby/cleaner.rb
#!/usr/bin/env ruby

def uniqueName
    return "kross_ruby_example"
end

def description
    return "##### KROSS RUBY EXAMPLE #####"
end

def iconName
    return "konqueror"
end

def youlaji
    return File.file?("/var/tmp/rubysaolaji.txt")
end

def saolaji
    File.delete("/var/tmp/rubysaolaji.txt")
    return true
end

File: examples/python/cleaner.py
#!/usr/bin/env python

import os.path

def uniqueName():
    return "kross_python_example"

def description():
    return "##### KROSS PYTHON EXAMPLE #####"

def iconName():
    return "konqueror"

def youlaji():
    return os.path.isfile("/var/tmp/pythonsaolaji.txt")

def saolaji():
    os.remove("/var/tmp/pythonsaolaji.txt")
    return True
